Makefile.toml

```diff
@@ -20,98 +20,38 @@ TLS_PROVIDER = "nativetls"
 [tasks.check]
 command = "cargo"
 args = [
-	"check",
-	"--features=${TLS_PROVIDER},http_server,cli",
-	"@@remove-empty(BUILD_FLAVOR)",
-	"@@remove-empty(TARGET)"
-	]
+        "check",
+        "--features=${TLS_PROVIDER},http_server,cli",
+        "@@remove-empty(BUILD_FLAVOR)",
+        "@@remove-empty(TARGET)"
+        ]
 cwd = "src/esthri"
 
 [tasks.build]
 command = "cargo"
 args = [
-	"build",
-	"--features=${TLS_PROVIDER},http_server,cli",
-	"@@remove-empty(BUILD_FLAVOR)",
-	"@@remove-empty(TARGET)"
-	]
+        "build",
```

Comment:
What tool did you use to format this Makefile.toml?

Reply:
This is hand formatted. but it would be nice if there was as toml "prettifier" tool out there, maybe there is?

Reply:
Might already be built in to cargo-make actually.
https://github.com/sagiegurari/cargo-make/blob/dc907c6e9ce41209115a4ee60db8b70f299f84fb/src/lib/descriptor/makefiles/toml.toml#L5

Reply:
🎉